validator-cli/src/ArbToEth/watcherArbToGnosis.ts

```
@@ -96,52 +96,44 @@
+  const amb = getAMB(gnosisAMBAddress, process.env.PRIVATE_KEY, process.env.RPC_GNOSIS);
 
   const wethAddress = (await retryOperation(() => veaOutbox.weth(), 1000, 10)) as string;
-  const weth = getWETHProvider(wethAddress, process.env.PRIVATE_KEY, providerGnosis);
-  const balance = (await retryOperation(() => weth.balanceOf(watcherAddress), 1000, 10)) as BigNumber;
-  const allowance = (await retryOperation(
-    () => weth.allowance(watcherAddress, veaOutboxAddress),
-    1000,
-    10
-  )) as BigNumber;
+  const weth = getWETH(wethAddress, process.env.PRIVATE_KEY, process.env.RPC_GNOSIS);
+  const balance = (await retryOperation(() => weth.balanceOf(watcherAddress), 1000, 10)) as BigInt;
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)